lua/cmp_luasnip/init.lua

```diff
@@ -98,4 +98,14 @@ function source:execute(completion_item, callback)
 	callback(completion_item)
 end
 
+function _G.__reload_completion_items()
+	local ft = require("luasnip.session").latest_load_ft
+	snip_cache[ft] = nil
```

Comment:
I don't think that would work with compound-filetypes, if `ft=lua.c` the cache stores items in `snip_cache["lua.c"]`, whereas this would set `snip_cache.lua` and `snip_cache.c` to nil.
I think the cache should be changed to store the items in `snip_cache.lua` and `snip_cache.c`.